### ISSUE TITLE
Add a material light-blue accent color (and/or why not custom accent color)

### ISSUE THREAD

**Issue by Guema** [material-theme/vsc-material-theme#584]
**Is your feature request related to a problem? Please describe.**
I like to use accent colors corresponding to the version icon color of Vscode i'm using :
- Insiders : Breaking Bad (green)
- Exploration : Orange
- Release : Not really ideal color (Blue/Indigo are to dark, Cyan is too greenish/light) 

**Describe the solution you'd like**
- Please add a light-blue variant from the material palette roughly matching Vscode release icon color.

**Describe alternatives you've considered**
- Alternatively you could add a custom color option in the menu, letting us switch all accent-themed targets with the input color (setting the transparency where it needs to be, like other accents)

**Additional context**
I took a basic light blue from material palette, and as you can see, it match pretty OK, not sure about how it would render with light theme and if it is the better one, as i'm not a designer, I think you will chose better than me ^^.
![image](https://user-images.githubusercontent.com/8943532/194516712-c37d8f8f-710a-44e2-a702-c4df41e2491c.png)

**Comment by equinusocio** [comment]
Have you tried Birght Teal?

**Comment by Guema** [comment]
Yes, i tried all available accent colors. Bright teal and "default" teal are more greenish than cyan, which is itself already too greenish for the vscode logo. (Btw there is alot of shades of cyan/teal, but no light blue, which would be cool)

The color i'm showing in this screenshot is a custom color, just to show that material light blue works quite ok with Vscode blue, but i had to edit each individual value with the one i wanted to try.

**Comment by equinusocio** [comment]
Cyan from Material Theme accents list. 
![CleanShot 2022-10-10 at 08 58 56](https://user-images.githubusercontent.com/10454741/194812522-bbc2eb05-2707-456a-b2f7-24a4740d079c.png)


Btw, at the moment there are not new accents planned. If you want to customize the accent by using your own color, you have to do the following:

1. [Go her](https://github.com/material-theme/vsc-material-theme/blob/master/scripts/generator/color-set.ts)e and look for any key using `theme.scheme.defaultAccent`
2. Copy the keys and add them in your local configuration file [overrides](https://github.com/material-theme/vsc-material-theme#color-scheme-override).

```json
"workbench.colorCustomizations": {
	"[Material Theme*]": {
		"COPIED KEY": "#ff0000",
	}
},
```
